modules/end-user-guide/partials/proc_creating-a-workspace-from-with-a-publicly-accessible-standalone-devfile-using-http.adoc

```diff
@@ -18,6 +18,6 @@ ifeval::["{project-context}" == "che"]
 .Example
 [subs="+quotes"]
 ----
-https://che.openshift.io/#https://gist.githubusercontent.com/themr0c/ef8e59a162748a8be07e900b6401e6a8/raw/8802c20743cde712bbc822521463359a60d1f7a9/devfile.yaml
+https://workspaces.openshift.com/#https://github.com/eclipse/che-docs
```

Comment:
@themr0c this is an incorrect reference to factory - should be based on `f?url=` see #1884